src/jira.py

```diff
@@ -91,6 +87,19 @@ def __init__(self, key: str = None, response: dict = None):
         if "customfield_10014" in response["fields"].keys():  # This custom field holds story point values
```

Comment:
Perhaps we can think about creating a class which encapsulates such rather cryptic _customfield_ names?

Reply:
Would be better than adding a comment (I find).